### ISSUE TITLE
Add more collaborators to the organization

### ISSUE THREAD

**Issue by aminya** [uncss/uncss#459]
Currently, the pull requests are left unattended for a long time, and many changes are still unreleased. To prevent the project from dying, @uncss should add more collaborators to the organization.

**Comment by aminya** [comment]
The latest version of Uncss is still depending on Postcss 7 that has severe security issues. If you cannot support this repository, please let us know so we make a fork.
https://github.com/parcel-bundler/parcel/issues/6315

**Comment by RyanZim** [comment]
As you can see, `master` has Postcss v8 support, but the only people with npm access are @giakki and @mikelambert, so there's nothing I can do without them

**Comment by mikelambert** [comment]
Hi folks, i am "still around" but also no longer use UnCSS (or even write JS at all!), and am not actively supporting UnCSS either.

Do we have any active contributors that we-the-community feel comfortable "owning" UnCSS, that others would vouch for? (Is this a matter of looking for "who has the most pull requests, and ask them"?) Some of us put in time for a couple years, but suspect we need to pass the torch at this point. @aminya do you have time/interest here yourself?

I do not have permissions to add any additional collaborators or transfer this repo, but we can reach out to @giakki (here or trying direct email) for help if necessary, as this is how we previously unblocked this (and perhaps ask for admin access this time).

Also, yes I do have npm access, and the ability to share npm access. I would prefer not to jump back in and merge PRs and do a release if i have no means to use/test anything (for fear of follow-up patches and more work), so would prefer to re-focus on the above github maintainer, and go from there.

**Comment by aminya** [comment]
> @aminya do you have time/interest here yourself?

Yes, I will be able to maintain the project by checking, testing, and merging the pull requests. People are usually [very active](https://github.com/uncss/uncss/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) in contributing to open source projects. We need someone that merges the PRs to complete the cycle.

**Comment by aminya** [comment]
@mikelambert Were you able to contact the owner?

**Comment by NovaAtWarren** [comment]
I know I haven't bee around for **that** long, only having one PR and all, but I'm willing to be a reviewer, help catch issues before they're merged, enforce code and naming style, stuff like that.

**Comment by tomap** [comment]
Any progress on this issue? Adding more owners? And be able to publish a new version on npm?

**Comment by NovaAtWarren** [comment]
In lieu of adding collaborators and maintainers, I've created a fork at [https://github.com/novaatwarren/uncss](https://github.com/novaatwarren/uncss) and [https://www.npmjs.com/package/@novaatwarren/uncss](https://www.npmjs.com/package/@novaatwarren/uncss).
I've merged a couple of PRs into it, and will merge more as they come in.
Please forgive me if it's not ran great near the beginning, as I'm not too familiar with the internals of uncss. I just _need_ to update uncss dependencies in one of my projects and this seems to be the only way for now.
